src/generateSdk/generateSdkApi.ts

```diff
@@ -55,12 +62,29 @@ export async function sdkGeneratorApiHandler(
             );
         }
 
+        if (astGeneratorOutput.program.body !== undefined && language === "ts") {
+            const astClassDefinition = astGeneratorOutput.program.body.find(
+                (node) =>
+                    node.type === AstNodeType.ClassDefinition &&
+                    (node as ClassDefinition).name === classConfiguration.name,
+            ) as ClassDefinition;
+            const currentInvalidHttpMethods = checkInvalidHttpMethods(
+                classConfiguration,
+                astClassDefinition,
+            );
+            invalidHttpMethods = invalidHttpMethods.concat(currentInvalidHttpMethods.methods);
+            invalidHttpMethodsString += currentInvalidHttpMethods.methodsString;
+        }
```

Comment:
As we discussed yesterday, I would perform this check in the bundling step. The SDK generation may not even happed if the user chooses not to, but the code still has to be correct.